doc/contributing/primordials.md

```diff
@@ -705,3 +705,27 @@ class SomeClass {
 ObjectDefineProperty(SomeClass.prototype, 'readOnlyProperty', kEnumerableProperty);
 console.log(new SomeClass().readOnlyProperty); // genuine data
 ```
+
+### Defining a `Proxy` handler
+
+When defining a `Proxy`, the handler object could be at risk of prototype
+pollution when using a plain object literal:
+
+```js
+// User-land
+Object.prototype.get = () => 'Unrelated user-provided data';
+
+// Core
+const objectToProxy = { someProperty: 'genuine value' };
+
+const proxyWithPlainObjectLiteral = new Proxy(objectToProxy, {
+  has() { return false; },
+});
+console.log(proxyWithPlainObjectLiteral.someProperty); // Unrelated user-provided data
```

Comment:
Totally out-of-scope for this PR, but usage of `console.log` in js code examples makes me wonder if a sort of comment convention deserves to be eventually invented and widely used within all Node.js docs. Maybe something like:
```mjs
proxyWithPlainObjectLiteral.someProperty //= 'Unrelated user-provided data'
//^ Unrelated user-provided data
//+ proxyWithPlainObjectLiteral.someProperty === 'Unrelated user-provided data'
//S proxyWithPlainObjectLiteral.someProperty: Unrelated user-provided data
console.log(proxyWithPlainObjectLiteral.someProperty); //> Unrelated user-provided data
```
I.e. shorthands for "evaluates to this value", "assert this", "returns following string", "prints this to stdout", "throws that", "expected this but failed", "returns this but expected something else", "the line {above,below} returns this", etc.

Reply:
The upside of using `console.log` is that readers don't have to trust the comment, they can copy/paste the code to verify for themselves.